build/npm/update-localization-extension.js

```diff
@@ -24,7 +24,7 @@ function update(options) {
 		throw new Error(`${location} doesn't exist.`);
 	}
 	let externalExtensionsLocation = options.externalExtensionsLocation;
-	if (externalExtensionsLocation !== undefined && !fs.existsSync(location)) {
+	if (externalExtensionsLocation !== undefined && !fs.existsSync(externalExtensionsLocation)) {
```

Comment:
this was a typo and an incorrect check before. I want to include this in the candidate because it is one line and is the correct check